R/paths.R

```diff
@@ -336,6 +323,12 @@ renv_paths_root_default_tempdir <- function() {
 #' \code{RENV_PATHS_MRAN}            \tab The path containing MRAN-related metadata. See `vignette("mran", package = "renv")` for more details. \cr
 #' }
 #'
+#' (If you want these settings to persist in your project, it is recommended that
+#' you add these to an appropriate \R startup file. For example, these could be
+#' set in: a project-local `.Renviron`, the user-level `.Renviron`, or a
```

Comment:
Use bullets?

Reply:
Again, I deliberately inlined a formerly bulleted list because otherwise I think it ends up taking up a lot of space on something that I think is mostly an aside because if the reader doesn't already know that they'll need to set up these env vars, this isn't enough context to help them.